src/repository/storage/postgres/orm/sequelize/editorTools.ts

```diff
@@ -70,14 +77,23 @@ export default class UserSequelizeStorage {
      */
     this.model = EditorToolModel.init({
       id: {
-        type: DataTypes.STRING,
+        type: DataTypes.INTEGER,
+        autoIncrement: true,
         allowNull: false,
         primaryKey: true,
       },
       name: {
         type: DataTypes.STRING,
         allowNull: false,
       },
+      author: {
+        type: DataTypes.INTEGER,
+        allowNull: true,
+        references: {
```

Comment:
why are we allowing null?

Reply:
because default tools have no creator for now